tsconfig.json

```diff
@@ -21,6 +21,11 @@
     "lib": ["dom", "esnext"],
     "preserveSymlinks": true
   },
-  "include": ["src/**/*.ts", "perf/**/*.ts", "perf/**/*.js"],
+  "include": [
+    "src/**/*.ts",
+    "perf/**/*.ts",
+    "perf/**/*.js",
+    "node_modules/@phritz/licensing/**/*.ts"
```

Comment:
This is not the usual way to do this.

The `@phritz/licensing` package includes the js and .d.ts files so TS knows how to include that.

...but I guess it is a private package so we could even use a github dependency in our package.json. Whatever is easiest.

Reply:
Apparently I can remove it, so I just did in the new PR.